unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -274,10 +298,13 @@ fn date_constraint(inverted: bool, mut node: Pairs<Rule>) -> RuleFragment {
     })
 }
 
-fn semver_constraint(inverted: bool, mut node: Pairs<Rule>) -> RuleFragment {
-    let context_getter = context_value(node.next().unwrap().into_inner());
-    let ordinal_operation = to_ordinal_comparator(node.next().unwrap());
-    let semver = semver(node.next().unwrap());
+fn semver_constraint(inverted: bool, node: Pairs<Rule>) -> RuleFragment {
+    let children: [Pair<Rule>; 3] = drain(node);
```

Comment:
Could be done in one line. I'll fix after review. Don't want to muck with the upstream branch